Karpach.RemoteShutdown.Controller/DelayForm.Designer.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Karpach.RemoteShutdown.Controller
+{
+    partial class DelayForm
+    {
+        /// <summary>
+        /// Required designer variable.
+        /// </summary>
+        private System.ComponentModel.IContainer components = null;
+
+        /// <summary>
+        /// Clean up any resources being used.
+        /// </summary>
+        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #region Windows Form Designer generated code
+
+        /// <summary>
+        /// Required method for Designer support - do not modify
+        /// the contents of this method with the code editor.
+        /// </summary>
+        private void InitializeComponent()
+        {
+            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(DelayForm));
+            this.btnAbort = new System.Windows.Forms.Button();
+            this.labelDelay = new System.Windows.Forms.Label();
+            this.SuspendLayout();
+            // 
+            // btnAbort
+            // 
+            this.btnAbort.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Right)));
+            this.btnAbort.DialogResult = System.Windows.Forms.DialogResult.Abort;
+            this.btnAbort.Location = new System.Drawing.Point(198, 64);
+            this.btnAbort.Margin = new System.Windows.Forms.Padding(4, 3, 4, 3);
+            this.btnAbort.Name = "btnAbort";
+            this.btnAbort.Size = new System.Drawing.Size(136, 27);
+            this.btnAbort.TabIndex = 4;
+            this.btnAbort.Text = "Abort the operation";
+            this.btnAbort.UseVisualStyleBackColor = true;
+            // 
+            // labelDelay
+            // 
+            this.labelDelay.AutoSize = true;
+            this.labelDelay.Location = new System.Drawing.Point(28, 33);
+            this.labelDelay.Name = "labelDelay";
+            this.labelDelay.Size = new System.Drawing.Size(38, 15);
+            this.labelDelay.TabIndex = 6;
+            this.labelDelay.Text = "label1";
+            // 
+            // DelayForm
+            // 
+            this.AutoScaleDimensions = new System.Drawing.SizeF(7F, 15F);
+            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
+            this.ClientSize = new System.Drawing.Size(347, 103);
+            this.Controls.Add(this.labelDelay);
+            this.Controls.Add(this.btnAbort);
+            this.FormBorderStyle = System.Windows.Forms.FormBorderStyle.None;
```

Comment:
Why System.Windows.Forms.FormBorderStyle.None? Visually it does not look good. I think it should be FixedDialog.